testdata/intake-v3/rum_events.ndjson

```diff
@@ -1,4 +1,14 @@
+{"y":{"id":"fc546e87a90a774f","x_id":"ec2e280be8345240","parent_id":"ec2e280be8345240","trace_id":"286ac3ad697892c406528f13c82e0ce1","n":"Parsing the document, executing sy. scripts","t":"hard-navigation","su":"browser-timing","s":14,"d":106}}
+{"y":{"id":"fb8f717930697299","x_id":"ec2e280be8345240","parent_id":"ec2e280be8345240","trace_id":"286ac3ad697892c406528f13c82e0ce1","n":"h://localhost:8000/test/e2e/general-usecase/app.e2e-bundle.min.js","t":"rc","su":"script","s":22.53499999642372,"d":35.060000023804605,"c":{"h":{"url":"h://localhost:8000/test/e2e/general-usecase/app.e2e-bundle.min.js?token=REDACTED","r":{"ts":677175,"ebs":676864,"dbs":676864}},"dt":{"se":{"n":"h://localhost:8000","rc":"localhost:8000","t":"rc"},"ad":"localhost","po":8000}}}}
+{"y":{"id":"9b80535c4403c9fb","x_id":"ec2e280be8345240","parent_id":"ec2e280be8345240","trace_id":"286ac3ad697892c406528f13c82e0ce1","n":"OpenTracing y","t":"cu","s":96.92999999970198,"d":198.07000000029802}}
+{"y":{"id":"5ecb8ee030749715","x_id":"ec2e280be8345240","parent_id":"ec2e280be8345240","trace_id":"286ac3ad697892c406528f13c82e0ce1","n":"GET /test/e2e/common/data.json","t":"external","su":"h","sy":true,"s":98.94000005442649,"d":6.72499998472631,"c":{"h":{"mt":"GET","url":"h://localhost:8000/test/e2e/common/data.json?test=hamid","sc":200},"dt":{"se":{"n":"h://localhost:8000","rc":"localhost:8000","t":"external"},"ad":"localhost","po":8000}}}}
+{"y":{"id":"27f45fd274f976d4","x_id":"ec2e280be8345240","parent_id":"ec2e280be8345240","trace_id":"286ac3ad697892c406528f13c82e0ce1","n":"POST h://localhost:8003/data","t":"external","su":"h","sy":true,"s":106.52000003028661,"d":11.584999971091747,"c":{"h":{"mt":"POST","url":"h://localhost:8003/data","sc":200},"dt":{"se":{"n":"h://localhost:8003","rc":"localhost:8003","t":"external"},"ad":"localhost","po":8003}}}}
+{"y":{"id":"a3c043330bc2015e","x_id":"ec2e280be8345240","parent_id":"ec2e280be8345240","trace_id":"286ac3ad697892c406528f13c82e0ce1","n":"POST h://localhost:8003/fetch","t":"external","su":"h","sy":false,"s":119.93500008247793,"d":15.949999913573265,"c":{"h":{"mt":"POST","url":"h://localhost:8003/fetch","sc":200},"dt":{"se":{"n":"h://localhost:8003","rc":"localhost:8003","t":"external"},"ad":"localhost","po":8003}}}}
+{"y":{"id":"bc7665dc25629379","x_id":"ec2e280be8345240","parent_id":"ec2e280be8345240","trace_id":"286ac3ad697892c406528f13c82e0ce1","n":"Fire \"DOMContentLoaded\" event","t":"hard-navigation","su":"browser-timing","s":120,"d":2}}
+{"me":{"x":{"n":"general-usecase-initial-p-load","t":"p-load"},"sa":{"xdc":{"v":1},"xds":{"v":295},"xbc":{"v":1}}}}
+{"me":{"x":{"n":"general-usecase-initial-p-load","t":"p-load"},"y":{"t":"Request"},"sa":{"ysc":{"v":1},"yss":{"v":1}}}}
+{"me":{"x":{"n":"general-usecase-initial-p-load","t":"p-load"},"y":{"t":"Response"},"sa":{"ysc":{"v":1},"yss":{"v":1}}}}
+{"me":{"x":{"n":"general-usecase-initial-p-load","t":"p-load"},"y":{"t":"Processing"},"sa":{"ysc":{"v":1},"yss":{"v":124}}}}
```

Comment:
One of the spans should contain a `stacktrace`. It also has been tested so far with the [rum_transactions+span.ndjon](https://github.com/elastic/apm-server/pull/3414/files#diff-e1b3950e4b04b62f51bd12d0829ab2a5L3)